### PR TITLE
Add examples

### PR DESCRIPTION
**Pull request by konradh**

### CODE REVIEW

File: examples/measurements/ed25519_measure.c
Comment (felix-gohla) [inline examples/measurements/ed25519_measure.c:20]: 🆗 This could be in the `main` function... ^^

File: examples/measurements/ed25519_measure.c
Comment (felix-gohla) [inline examples/measurements/ed25519_measure.c:17]: Why are they outside the main function and if so, why not `static`?

File: examples/measurements/inflate_measure.c
Comment (felix-gohla) [inline examples/measurements/inflate_measure.c:19]: Same as above.

File: examples/measurements/sha256_measure.c
Comment (felix-gohla) [inline examples/measurements/sha256_measure.c:17]: `static`?

File: examples/measurements/sha512_measure.c
Comment (felix-gohla) [inline examples/measurements/sha512_measure.c:17]: `static`?

File: examples/measurements/aes_gcm_measure.c
Comment (felix-gohla) [inline examples/measurements/aes_gcm_measure.c:20]: `static`?

File: examples/measurements/inflate_measure.c
Comment (felix-gohla) [inline examples/measurements/inflate_measure.c:14]: This is defined in `gpio.c` and doesn't have an effect here, does it?

File: examples/measurements/sha256_measure.c
Comment (felix-gohla) [inline examples/measurements/sha256_measure.c:12]: This is defined in `gpio.c` and doesn't have an effect here, does it?

File: examples/measurements/sha512_measure.c
Comment (felix-gohla) [inline examples/measurements/sha512_measure.c:12]: Defined in `gpio.c`.

**Comment by felix-gohla** [comment]
Please do not merge until #19 is merged.

**Review by felix-gohla — CHANGES_REQUESTED** [review]

File: examples/measurements/ed25519_measure.c
Comment (konradh) [reply examples/measurements/ed25519_measure.c]: I made these constants `static`, as requested, and also commented out unused private keys and expected values. I did not use `memset` and did not move them into `main`, because I think constants should not be defined in functions. If you really want them in `main`, maybe react with :-1: and I'll change it :)

Comment (felix-gohla) [reply examples/measurements/ed25519_measure.c]: I would argue that this is less readable than a memset and globals are not always a good thing (especially if they are only used in this function). However as we did it in all other measurement files like this, too, I'd be fine with leaving it as is. ^^

**Comment by felix-gohla** [comment]
Is this a thing of Github showing me 25 commits (some are from #19)? Not sure, what's happening here. 😊

**Review by felix-gohla — APPROVED** [review]

**Pull request merged**
